python/tvm/relay/qnn/op/qnn.py

```diff
@@ -72,3 +72,50 @@ def requantize(data,
                             output_zero_point,
                             rounding,
                             out_dtype)
+
+
+def quantize(input_data, output_zero_point, output_scale, out_dtype='int8'):
+    r""" Quantize op
+     This operator takes float32 as input and produces quantized int8 or unit8 as output.
+     The input tensor can be of any shape. The output shape is the same as input shape.
+     ..math::
+            \mbox{out}[x] =
+                \mbox{clamp(round(input_tensor/output_scale) + output_zero_point);
+                 out_dtype::min, out_dtype::max}
+     Parameters
+    ----------
+    input_data : tvm.relay.Expr
+        The input tensor to be quantized. Can be of type float32.
+    output_zero_point :
+        The output zero_point.
+    output_scale:
+        The output scale.
+    input_dtype:
+        The data type of the input tensor. Can be [int8, uint8]
+    Returns
+    -------
+    result : tvm.relay.Expr
+        The computed result.
+    """
+    return _make.quantize(input_data, output_zero_point, output_scale, out_dtype)
+
+
+def dequantize(input_data, input_zero_point, input_scale):
```

Comment:
Same comments for the dequantize as for quantize
* scale and zero point reordering
* missing types in the description